test/functional/tests/incremental_load/test_inactive_cores.py

```diff
@@ -16,152 +18,192 @@
-        3. Remove one of core devices.
-        4. Load cache.
-        5. Check if cache has appropriate number of valid and inactive core devices.
+    title: Test for inactive core configuration statistics.
+    description: |
+        Test the cache inactive core configuration statistics after removing one of core devices
+        and loading cache.
+    pass_criteria:
+      - Cache can be loaded with inactive core device.
+      - CAS correctly reports inactive core statistics in cache configuration statistics after
+      loading cache.
     """
-    cache, core_device = prepare()
+    core_number = 3
+
+    with TestRun.step("Prepare cache and core devices"):
+        cache_device = TestRun.disks["cache"]
+        core_device = TestRun.disks["core"]
+
+        cache_device.create_partitions([Size(500, Unit.MebiByte)])
+        core_device.create_partitions([Size(1, Unit.GibiByte)] * core_number)
 
-    cache_device = cache.cache_device
-    stats = cache.get_statistics()
+        cache_device = cache_device.partitions[0]
+        core_device_partitions = core_device.partitions
 
-    assert stats.config_stats.core_dev == 3
-    assert stats.config_stats.inactive_core_dev == 0
+    with TestRun.step("Start cache"):
+        cache = casadm.start_cache(cache_device, force=True)
 
-    TestRun.LOGGER.info("Stopping cache")
-    cache.stop()
+    with TestRun.step("Add cores to the cache"):
+        for core_device_part in core_device_partitions:
+            cache.add_core(core_dev=core_device_part)
 
-    TestRun.LOGGER.info("Removing one of core devices")
-    core_device.remove_partitions()
-    core_device.create_partitions([Size(1, Unit.GibiByte), Size(1, Unit.GibiByte)])
+    with TestRun.step("Check if correct number of inactive cores is displayed in cache statistics"):
+        stats = cache.get_statistics()
+        if stats.config_stats.inactive_core_devices != 0:
+            TestRun.fail("Inactive core in statistics after starting cache")
 
-    TestRun.LOGGER.info("Loading cache with missing core device")
-    cache = casadm.start_cache(cache_device, load=True)
-    stats = cache.get_statistics()
+    with TestRun.step("Stop cache"):
+        cache.stop()
 
-    assert stats.config_stats.core_dev == 3
-    assert stats.config_stats.inactive_core_dev == 1
+    with TestRun.step("Remove last core device"):
+        core_device.remove_partition(part=core_device_partitions[-1])
+
+    with TestRun.step("Load cache with missing core device"):
+        cache = casadm.start_cache(cache_device, load=True)
+
+    with TestRun.step(
+        "Check if correct number of cores and inactive cores is displayed in cache statistics"
+    ):
+        stats = cache.get_statistics()
+        if stats.config_stats.core_dev != 3:
+            TestRun.fail("Wrong number of cores after loading the cache")
+        if stats.config_stats.inactive_core_devices != 1:
+            TestRun.fail("Wrong number of inactive cores after loading the cache")
```

Comment:
Maybe it's worth to print statistics or amount of cores in this message (same for line 70)?